examples/api_client.py

```diff
@@ -1,72 +1,77 @@
 
-import xmlrpclib
 import json
-import time
+import xmlrpclib
 
 if __name__ == '__main__':
 
     api = xmlrpclib.ServerProxy("http://bradley:password@localhost:8442/")
 
-    print 'Let\'s test the API first.'
+    print('Let\'s test the API first.')
     inputstr1 = "hello"
     inputstr2 = "world"
-    print api.helloWorld(inputstr1, inputstr2)
-    print api.add(2,3)
+    print(api.helloWorld(inputstr1, inputstr2))
+    print(api.add(2, 3))
 
-    print 'Let\'s set the status bar message.'
-    print api.statusBar("new status bar message")
+    print('Let\'s set the status bar message.')
+    print(api.statusBar("new status bar message"))
 
-    print 'Let\'s list our addresses:'
-    print api.listAddresses()
+    print('Let\'s list our addresses:')
+    print(api.listAddresses())
 
-    print 'Let\'s list our address again, but this time let\'s parse the json data into a Python data structure:'
+    print('Let\'s list our address again, but this time let\'s parse the json data into a Python data structure:')
     jsonAddresses = json.loads(api.listAddresses())
-    print jsonAddresses
-    print 'Now that we have our address data in a nice Python data structure, let\'s look at the first address (index 0) and print its label:'
-    print jsonAddresses['addresses'][0]['label']
-
-    print 'Uncomment the next two lines to create a new random address with a slightly higher difficulty setting than normal.'
-    #addressLabel = 'new address label'.encode('base64')
-    #print api.createRandomAddress(addressLabel,False,1.05,1.1111)
-
-    print 'Uncomment these next four lines to create new deterministic addresses.'
-    #passphrase = 'asdfasdfqwser'.encode('base64')
-    #jsonDeterministicAddresses = api.createDeterministicAddresses(passphrase, 2, 4, 1, False)
-    #print jsonDeterministicAddresses
-    #print json.loads(jsonDeterministicAddresses)
-
-    #print 'Uncomment this next line to print the first deterministic address that would be generated with the given passphrase. This will Not add it to the Bitmessage interface or the keys.dat file.'
-    #print api.getDeterministicAddress('asdfasdfqwser'.encode('base64'),4,1)
-
-    #print 'Uncomment this line to subscribe to an address. (You must use your own address, this one is invalid).'
-    #print api.addSubscription('2D94G5d8yp237GGqAheoecBYpdehdT3dha','test sub'.encode('base64'))
-
-    #print 'Uncomment this line to unsubscribe from an address.'
-    #print api.deleteSubscription('2D94G5d8yp237GGqAheoecBYpdehdT3dha')
-
-    print 'Let\'s now print all of our inbox messages:'
-    print api.getAllInboxMessages()
+    print(jsonAddresses)
+    print('Now that we have our address data in a nice Python data structure,'
+          ' let\'s look at the first address (index 0) and print its label:')
+    print(jsonAddresses['addresses'][0]['label'])
+
+    print('Uncomment the next two lines to create a new random address'
+          ' with a slightly higher difficulty setting than normal.')
```

Comment:
As for me, this `print` makes no sense: it tells about the comment line which user doesn't see. Maybe for that reason it was commented (^ plenty of them ahead (8

Reply:
just to unclutter the screen at runtime and keep syntax hilite